app/models/manageiq/providers/vmware/infra_manager/host_esx.rb

```diff
@@ -177,4 +201,11 @@ def thumbprint_sha1
     require 'VMwareWebService/esx_thumb_print'
     ESXThumbPrint.new(ipaddress, authentication_userid, authentication_password).to_sha1
   end
+
+  private
+
+  def validate_active_with_power_state(feature, expected_power_state)
+    return unsupported_reason_add(feature, _("The Host is not connected to an active Provider"))   unless has_active_ems?
+    return unsupported_reason_add(feature, _("The host is not powered '#{expected_power_state}'")) unless expected_power_state == power_state
```

Comment:
"This host is not powered 'maintenance'" is going to read weird.  Not sure how else to do this though.